ssl_certificate_expansion.md

```diff
@@ -0,0 +1,66 @@
+# SSL Certificate expansion for Galaxy EU
+
+This README outlines the process for expanding the Galaxy EU SSL certificate to include a new wildcard domain, for example: `*.aquainfra.usegalaxy.eu`.
+
+# Using Route 53 and DNS challenge
+
+## Step 1: Create a Backup
+
+Before making any changes, it's essential to create a backup of the current SSL certificates
+
+1. `/etc/letsencrypt/live/usegalaxy/`
+2. `/etc/ssl/certs/fullchain.pem`
+3. `/etc/ssl/user/privkey-nginx.pem`
+
+## Step 2: Expand the Current Certificate
+
+Run the following command to expand the current certificate with the new domain `*.aquainfra.usegalaxy.eu`. This command should be executed as the root user, and AWS credentials are assumed to be available in the root's home directory (see [here](https://certbot-dns-route53.readthedocs.io/en/stable/) for details on AWS creds).
```

Comment:
```suggestion
Run the following command to expand the current certificate with the new domain `*.aquainfra.usegalaxy.eu`. This command should be executed as the root user, and AWS credentials are assumed to be available in the root's home directory (see [here](https://certbot-dns-route53.readthedocs.io/en/stable/) for details on AWS creds). AWS DNS is used beause is very easily scriptable. 
```

Reply:
it's also because it's basically the only way to get a wildcard certificate without running your own DNS server. (ok Cloudflare and some others with APIs are options)

Reply:
Updated.